testdata/source_json.json

```diff
@@ -0,0 +1,272 @@
+[
```

Comment:
let's use a shorter example that doesn't look like someone's personal information